src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Members.cs

```diff
@@ -143,7 +143,10 @@ public override void VisitProperty(IPropertySymbol symbol)
 
                 AddCustomModifiersIfRequired(symbol.RefCustomModifiers);
 
-                symbol.Type.Accept(this.NotFirstVisitor);
+                var symbolType = ShouldBeNullableAnnotated(symbol, FlowAnalysisAnnotations.MaybeNull)
+                    ? symbol.Type.WithNullableAnnotation(CodeAnalysis.NullableAnnotation.Annotated)
+                    : symbol.Type;
+                symbolType.Accept(this.NotFirstVisitor);
```

Comment:
This is one possible solution to append the question mark. I don't know whether that's the right approach for the problem.